.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -0,0 +1,58 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+<!--
+DO NOT FILE ISSUES FOR GENERAL SUPPORT QUESTIONS.
```

Comment:
Honestly, I don't mind handling support questions here. We can add this section back in if the number of issues blows up.